im2latex/trainers/visionencoderdecodertrainer.py

```diff
@@ -0,0 +1,332 @@
+from loguru import logger

```

Comment:
We should add support for weights and biases to keep track of the model's training remotely.
I can add this later without much of a problem.

Reply:
True